Index: src/main/java/com/example/tfp/Model/PublicMatch.java
===================================================================
diff --git a/src/main/java/com/example/tfp/Model/PublicMatch.java b/src/main/java/com/example/tfp/Model/PublicMatch.java
deleted file mode 100644
--- a/src/main/java/com/example/tfp/Model/PublicMatch.java	(revision 44667d5a731ad384c88b0d8620d6498e502fa962)
+++ /dev/null	(revision 44667d5a731ad384c88b0d8620d6498e502fa962)
@@ -1,26 +0,0 @@
-package com.example.tfp.Model;
-
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Id;
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
-
-@Setter
-@Getter
-@Entity
-@AllArgsConstructor
-@NoArgsConstructor
-public class PublicMatch {
-
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Integer id;
-
-    pr
-
-
-}
