Index: src/main/java/com/example/tfp/Model/PublicMatch.java
===================================================================
diff --git a/src/main/java/com/example/tfp/Model/PublicMatch.java b/src/main/java/com/example/tfp/Model/PublicMatch.java
deleted file mode 100644
--- a/src/main/java/com/example/tfp/Model/PublicMatch.java	(revision 15294e2812077d332841fc5dd0d114e898e421ce)
+++ /dev/null	(revision 15294e2812077d332841fc5dd0d114e898e421ce)
@@ -1,4 +0,0 @@
-package com.example.tfp.Model;
-
-public class PublicMatch {
-}
